code/modules/cargo/blackmarket/blackmarket_items/weapons.dm

```diff
@@ -352,12 +352,12 @@
 
 /datum/blackmarket_item/weapon/fireaxe
 	name = "Fire Axe"
-	desc = "An incredibly sharp axe of dubious make used by various engineers throughout the years to settle arguments while hammered. Sold to us by a very friendly man in a suit."
+	desc = "An incredibly sharp axe of reputable make, used by various engineers to settle arguments while hammered. Sold to us by a very friendly man in a suit."
 	item = /obj/item/melee/axe/fire
 
-	price_min = 2500
-	price_max = 3500
-	stock = 2
+	price_min = 1200
+	price_max = 2300
+	stock_max = 3
```

Comment:
```suggestion
	price_min = 1500
	price_max = 2500
	stock_max = 2
```
these values still feel really low to me but it will take them.

Reply:
I still feel like it's fundamentally a wielded melee weapon / tool and shouldn't be competing with automatic weapons price-wise